listener/listener.go

```diff
@@ -320,8 +320,13 @@ func ensureSubscription(ctx context.Context, sm *servicebus.SubscriptionManager,
 	if err == nil {
 		return subEntity, nil
 	}
-
-	return sm.Put(ctx, name)
+	increaseMaxDelivery := func(s *servicebus.SubscriptionDescription) error {
+		var maxDeliveryCount int32 = 500
```

Comment:
I don't think hardcoding this as the final solution is a good idea. If we take this approach I think we should expose setting maxDeliveryCount as a setting to the user

Reply:
Made configurable.